plugins/source/aws/resources/services/iam/policies_fetch.go

```diff
@@ -41,3 +42,15 @@ func resolveIamPolicyTags(ctx context.Context, meta schema.ClientMeta, resource
 	}
 	return resource.Set("tags", client.TagsToMap(response.Tags))
 }
+
+func resolveIamPolicyVersionList(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	r := resource.Item.(types.ManagedPolicyDetail)
+	for i := range r.PolicyVersionList {
+		if v, err := url.PathUnescape(aws.ToString(r.PolicyVersionList[i].Document)); err == nil {
+			r.PolicyVersionList[i].Document = &v
+		} else {
+			meta.Logger().Warn().Err(err).Msg("Failed to unescape policy document, leaving as-is")
```

Comment:
nit: We may want to put some information about the policy ID or something in this log message, so it's possible to locate & fix it.

Reply:
added